pyproject.toml

```diff
@@ -48,6 +48,7 @@ pytest-timeout = "2.2.0"
 [tool.pytest.ini_options]
 filterwarnings = [
     "ignore::DeprecationWarning:pkg_resources",
+    "ignore::DeprecationWarning",
```

Comment:
I'm not sure if we want to ignore 'all' deprecation warnings.

Reply:
Was testing something - will remove